packages/ember-metal/lib/index.js

```diff
@@ -275,30 +275,6 @@ Ember.isGlobalPath = isGlobalPath;
 
 Ember.run = run;
 
-/**
-@class Backburner
-@for Ember
-@private
-*/
-Ember.Backburner = function() {
-  deprecate(
-    'Usage of Ember.Backburner is deprecated.',
-    false,
-    {
-      id: 'ember-metal.ember-backburner',
-      until: '2.8.0',
```

Comment:
Were you going to cherry pick this to the beta branch? The reason why I removed it is master should be the next LTS.